backend/clubs/serializers.py

```diff
@@ -745,6 +745,7 @@ class Meta:
             "founded",
             "image_url",
             "is_favorite",
+            "favorite_count",
```

Comment:
Why was this line added? Not sure if good or bad, just curious.

Reply:
accidentally deleted it

Reply:
oh wait turns out it is redundant. I think rebasing might have fixed my deletion. will recommit without.